cropduster/tests/test_models.py

```diff
@@ -34,6 +39,7 @@ def test_original_image(self):
         self.assertIs(article.lead_image.sizes, Article.LEAD_IMAGE_SIZES)
 
     def test_generate_thumbs(self):
+        reload(cropduster_settings)
```

Comment:
I know this isn't up for code review yet, but I think what you actually want to use here is the [`setting_changed`](https://docs.djangoproject.com/en/3.0/ref/signals/#setting-changed) signal. Here is an example of its usage from django-pipeline:

https://github.com/jazzband/django-pipeline/blob/ce19344ef414770753b2d8ae310e6d778dc3599e/pipeline/conf.py#L121-L124

```python
@receiver(setting_changed)
def reload_settings(**kwargs):
    if kwargs['setting'] == 'PIPELINE':
        settings.update(kwargs['value'])
```

In truth, it's an anti-pattern to grab these settings and set them in a module, but short of a refactor of how settings are referenced internally in the app, the simplest option is to put a receiver like this in cropduster/settings.py

Reply:
Cool, i'll give that a try 👍

Reply:
With the caveat that instead of a settings dict you would have `global CROPDUSTER_CREATE_THUMBS` etc etc

Reply:
Gave this a go, more-or-less worked :+1: But (I think) because of the way we actually read this (and other settings) off `cropduster_settings`, the global method doesn't work. Patching it directly like so seemed to do the trick, though-
```
 @receiver(setting_changed)                                                                                            
 def reload_settings(**kwargs):                                                                                        
     if kwargs['setting'] == 'CROPDUSTER_CREATE_THUMBS':                                                               
         from cropduster import settings as cropduster_settings                                                        
         cropduster_settings.CROPDUSTER_CREATE_THUMBS = kwargs['value']    
```